frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -200,6 +205,9 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
     return _.intersection(existingAPIs, providedAPIs);
   };
 
+  const suggestedNamespaceRequestsMonitoring = (namespace) =>
+    namespace?.metadata?.labels?.['openshift.io/cluster-monitoring'] === 'true';
```

Comment:
We shouldn't be creating role bindings in the namespace based on an existing label. The user has to opt-in to monitoring.

Reply:
OK, will just create the `Role`